CHANGELOG.md

```diff
@@ -8,6 +8,19 @@
 
 <!-- towncrier release notes start -->
 
+## 16.7.0 (unreleased)
+
+### Breaking
+
+### Feature
+
+### Bugfix
+- Fix the URI validation Regex in `widgetValidation` component of `FormValidation.js` to support the relative URLs.
+
+### Internal
+
+### Documentation
+
 ## 16.6.0 (2022-12-23)
 
 ### Feature
```

Comment:
Please revert changes to this file. We use `towncrier`, and the file `3183.bugfix` will be used to update `CHANGELOG.md` automatically when a release is cut.

Reply:
Thank you! Done.